File: src/worker/core/isFinals.java
import db.idb;
import util.g;

public class FinalsChecker {
    public static async Task<Boolean> isFinals() {
        int[] numGamesPlayoffSeries = g.get("numGamesPlayoffSeries", "current");
        PlayoffSeries playoffSeries = await idb.cache.playoffSeries.get(g.get("season"));

        return playoffSeries != null && playoffSeries.currentRound == numGamesPlayoffSeries.length - 1;
    }
}

File: src/worker/core/isFinals.cpp
#include <iostream>
#include <string>
#include <optional>
#include <vector>

// Assuming necessary includes and using directives for idb and g

bool isFinals() {
    int numGamesPlayoffSeries = g.get("numGamesPlayoffSeries", "current");
    std::optional<PlayoffSeries> playoffSeries = idb.cache.playoffSeries.get(g.get("season"));

    return playoffSeries.has_value() && playoffSeries->currentRound == numGamesPlayoffSeries - 1;
}
